python.d/proxysql.chart.py

```diff
@@ -0,0 +1,349 @@
+    return 'SELECT {params} FROM {table}'.format(table=table, params=', '.join(params))
+
+
+# https://github.com/sysown/proxysql/blob/master/doc/admin_tables.md#stats_mysql_global
+QUERY_GLOBAL = query("stats_mysql_global",
+                     "Variable_Name",
+                     "Variable_Value",
+                     )
+
+# https://github.com/sysown/proxysql/blob/master/doc/admin_tables.md#stats_mysql_connection_pool
+QUERY_CONNECTION_POOL = query("stats_mysql_connection_pool",
+                              "hostgroup",
+                              "srv_host",
+                              "srv_port",
+                              "status",
+                              "ConnUsed",
+                              "ConnFree",
+                              "ConnOK",
+                              "ConnERR",
+                              "Queries",
+                              "Bytes_data_sent",
+                              "Bytes_data_recv",
+                              "Latency_us",
+                              )
+
+# https://github.com/sysown/proxysql/blob/master/doc/admin_tables.md#stats_mysql_commands_counters
+QUERY_COMMANDS = query("stats_mysql_commands_counters",
+                       "Command",
+                       "Total_Time_us",
+                       "Total_cnt",
+                       "cnt_100us",
+                       "cnt_500us",
+                       "cnt_1ms",
+                       "cnt_5ms",
+                       "cnt_10ms",
+                       "cnt_50ms",
+                       "cnt_100ms",
+                       "cnt_500ms",
+                       "cnt_1s",
+                       "cnt_5s",
+                       "cnt_10s",
+                       "cnt_INFs",
+                       )
+
+GLOBAL_STATS = [
+    'client_connections_aborted',
+    'client_connections_connected',
+    'client_connections_created',
+    'client_connections_non_idle',
+    'proxysql_uptime',
+    'questions',
+    'slow_queries']
+
+CONNECTION_POOL_STATS = [
+    'status',
+    'connused',
+    'connfree',
+    'connok',
+    'connerr',
+    'queries',
+    'bytes_data_sent',
+    'bytes_data_recv',
+    'latency_us']
+
+ORDER = [
+    'connections',
+    'active_transactions',
+    'questions',
+    'pool_overall_net',
+    'commands_count',
+    'commands_duration',
+    'pool_status',
+    'pool_net',
+    'pool_queries',
+    'pool_latency',
+    'pool_connection_used',
+    'pool_connection_free',
+    'pool_connection_ok',
+    'pool_connection_error']
+
+HISTOGRAM_ORDER = [
+    '100us',
+    '500us',
+    '1ms',
+    '5ms',
+    '10ms',
+    '50ms',
+    '100ms',
+    '500ms',
+    '1s',
+    '5s',
+    '10s',
+    'inf',
+]
+
+STATUS = {
+    "ONLINE": 1,
+    "SHUNNED": 2,
+    "OFFLINE_SOFT": 3,
+    "OFFLINE_HARD": 4
+}
+
+CHARTS = {
+    'pool_status': {
+        'options': [None, 'ProxySQL Backend Status', 'status', 'status', 'proxysql.pool_status', 'line'],
+        'lines': [
+        ]},
```

Comment:
With the recent PR's, all of the `CHARTS` entries have been updated to look like this:

```
    'pool_status': {
        'options': [None, 'ProxySQL Backend Status', 'status', 'status', 'proxysql.pool_status', 'line'],
        'lines': []
    },
```

With the closing bracket for the whole definition on it's own line, and the empty list definition for the `'lines'` key on a single line.  Please update to match that.

Reply:
Done :)